client/src/components/layouts/LayoutEpsilon/style.scss

```
@@ -0,0 +1,496 @@
+$alertLeft: 90%;
+$alertWidth: 5%;
+$alertPadding: 0.2%;
+$alertColor: #0f0;
+$navigatorWidth: 220px;
+
+$buttonBG: #0f1216;
+$boxBG: #151820;
+
+$alert5: #1F252D;
+
+$alert4: #3a6dda;
+$alert3: #acac34;
+$alert2: #cc7926;
+$alert1: #ca2a2a;
+$alertp: #4d1980;
+
+.epsilon-frame {
+  //font-family: Sarpanch;
+  font-family: Bebas Neue;
+  font-size: 1.2rem;
+  text-transform: uppercase;
+  width: 100vw;
+  height: 100vh;
+  position: relative;
+  background: #1F252D;
+  background-image: url("./background-crosses.png");
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  color: white;
+
+  small {
+	font-size:1rem;
+  }
+  
+  strong {font-weight:normal}
+  .card-sickbay .bunk.card {
+  color:#ccc;
+  font-style: italic;
+  strong {
+	padding-left: 10px; 
+	//padding-right: 2px; 
+	font-weight:normal;
+	font-style: normal;
+	color:white;
+	}
+  }
+  
+  // make sure text doesn't shift other columns
+  .targeting-control p{ 
+	  min-width:11ch;
+	
+	strong {
+	  font-weight:normal;
+	  padding-left:3ch;
+	  color:#ccc;
+	}
+	.text-danger{
+	min-width:9ch;
+	}
+  }
+
+  .frame-holder {
+    width: 100%;
+    height: 100%;
+  }
+
+  .left-frame {
+    width: 1%;
+    height: 100%;
+    position: absolute;
+    background-image: linear-gradient(
+      to right,
+      rgba(31, 37, 45, 0) 0%,
+      rgba(31, 37, 45, 0.85) 100%
+    );
+    }
+  .right-frame {
+    width: 1%;
+    height: 100%;
+    position: absolute;
+    right: 0px;
+    background-image: linear-gradient(
+      to left,
+      rgba(31, 37, 45, 0) 0%,
+      rgba(31, 37, 45, 0.85) 100%
+    );
+
+  }
+  .navigation {
+    position: absolute;
+    z-index: 1000000;
+    width: 100vw;
+    top: 0;
+    left: 0;
+    h1 {
+      padding: 0;
+      margin: 0;
+      position: absolute;
+      font-weight: 400;
+      font-size: 40px;
+      color: white;
+      font-family: "Commador";
+      font-style: italic;
+      text-transform: uppercase;
+      line-height: 1.5;
+    }
+    .card-name {
+      top: 10px;
+      font-style:normal;
+      font-family: "Bebas Neue";
+      //left: 2%;
+      right: 2%;
+      padding-left: 20px;
+      padding-right: 30px;
+      font-size: 24px;
+      cursor: pointer;
+      pointer-events: all;
+      border: 2px solid #777;
+      border-radius: 1em;
+      background-color: $buttonBG;
+      width:25ch;
```

Limiting the width of the card name and card list is probably not necessary, since they have plenty of room to grow for large card names. I think having the card names be on one line instead of wrapping would be a better appearance. I would change this as follows:

```suggestion
      width:auto;
```